Lib/test/test_perf_profiler.py

```diff
@@ -71,11 +62,13 @@ def baz():
                 """
         with temp_dir() as script_dir:
             script = make_script(script_dir, "perftest", code)
+            env = {**os.getenv(), "PYTHON_JIT": "0"}
```

Comment:
`os.getenv()` wants a key for environment variable to get. I think what you want is

```
env =  {**os.environ, "PYTHON_JIT": "0"}
```

Reply:
Lol yes 🤦 

man the jet lag is not good 😓